build.gradle

```
@@ -1,13 +1,13 @@
 plugins {
-  id 'org.springframework.boot' version '3.0.0'
+  id 'org.springframework.boot' version '2.7.1'
```

why not 2.7.7.

```suggestion
  id 'org.springframework.boot' version '2.7.7'
```

https://mvnrepository.com/artifact/org.springframework.boot/spring-boot